docs/remote_inference_blueprints/ocigenai_connector_chat_blueprint.md

```diff
@@ -0,0 +1,151 @@
+### OCIGenai Chat Connector Blueprint:
+
+This blueprint will show you how to connect a OCI Genai chat model to your Opensearch cluster.
+
+See [OCI GenAI Inference API docs](https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/api/#/en/generative-ai-inference/20231130/) for more details.
+
+#### 1. Create your Model connector and Model group
+
+##### 1a. Register Model group
+
+```json
+POST /_plugins/_ml/model_groups/_register
+
+{
+  "name": "ocigenai_model_group",
+  "description": "Your OCI Genai model group"
+}
+```
+
+This request response will return the `model_group_id`, note it down.
+
+##### 1b. Create Connector
+
+Refer to the [OCI GenAI Inference API docs](https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/api/#/en/generative-ai-inference/20231130/) for more parameters. This API request will create a connector pointing to OCI Genai API
+
+Later, when calling the newly created Predict action, you will pass `prompt` and any extra parameters you've included to the request.
+
+For authentication and authorization, you need to specify several parameters (Visit [OCI SDK Authentication Methods](https://docs.public.oneportal.content.oci.oraclecloud.com/en-us/iaas/Content/API/Concepts/sdk_authentication_methods.htm) for information)
+- auth_type: The type of authentication for OCI services, we support resource principal, instance principal, and user principal. 
+- tenant_id: the customer tenancy ID (required for user principal authentication)
+- user_id: the user ID (required for user principal authentication)
+- fingerprint: the user fingerprint (required for user principal authentication)
+- pemfile_path: the user private key path (required for user principal authentication)
+- region: the OCI region (required for user principal authentication)
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
```

Comment:
Why no `credential` field?

Reply:
Yeah for now the credentials is empty, since for OCI SDK credentials are provided as a file staying in the host, for example for user principal we need to provide the private key file path which we can just put as parameters